### PR TITLE
Update article: "Quick Test: Navigate using just your keyboard"

### PR DESCRIPTION
**Pull request by missmatsuko**
See issue #631

Open for comments/review.

### CODE REVIEW

File: _posts/2013-01-11-navigate-using-just-your-keyboard.md [hunk 1]
```diff
@@ -9,11 +9,26 @@ categories:
   - Quick Tests
 ---
 
-One quick way that you can check your accessibility is to navigate through the site using nothing but your keyboard.
+### Introduction
+Testing your site for keyboard navigation will ensure that keyboard users can access your site.
```
Comment (scottaohara) [inline _posts/2013-01-11-navigate-using-just-your-keyboard.md]: ...will *help* ensure...

File: _posts/2013-01-11-navigate-using-just-your-keyboard.md [hunk 2]
```diff
@@ -9,11 +9,26 @@ categories:
...
 
-1. Un-plug/turn off your mouse.
-2. Use the <kbd>Tab</kbd> key browse.
-3. Use <kbd>Enter</kbd> to select elements.
-4. Some menus and form elements may require <kbd>&uarr;</kbd> <kbd>&darr;</kbd> <kbd>&larr;</kbd> <kbd>&rarr;</kbd> arrow keys.
+As a bonus, sites that can be navigated with a keyboard can also likely be navigated using other input devices.
```
Comment (scottaohara) [inline _posts/2013-01-11-navigate-using-just-your-keyboard.md]: "can also likely" is an affirmative followed by a maybe.

"are more likely to be navigable"

would be safer a wording.

File: _posts/2013-01-11-navigate-using-just-your-keyboard.md [hunk 3]
```diff
@@ -9,11 +9,26 @@ categories:
...
 
-This is by no means a complete accessibility check, but it can quickly help you find major accessibility problems.
+### How to navigate a website with a keyboard
+Generally, pressing `tab` moves focus between elements and pressing `enter` will interact with them.
```
Comment (scottaohara) [inline _posts/2013-01-11-navigate-using-just-your-keyboard.md]: ...moves focus between *interactive* elements... 

use `<kbd>...</kbd>` to surround keys.

File: _posts/2013-01-11-navigate-using-just-your-keyboard.md [hunk 4]
```diff
@@ -9,11 +9,26 @@ categories:
   - Quick Tests
 ---
 
-One quick way that you can check your accessibility is to navigate through the site using nothing but your keyboard.
+### Introduction
```
Comment (scottaohara) [inline _posts/2013-01-11-navigate-using-just-your-keyboard.md]: It's implied that the paragraph immediately after an `h1` could be introductory. I don't think we need to call this out with a new heading.

File: _posts/2013-01-11-navigate-using-just-your-keyboard.md [hunk 5]
```diff
@@ -9,11 +9,26 @@ categories:
...
 
-This is by no means a complete accessibility check, but it can quickly help you find major accessibility problems.
+### How to navigate a website with a keyboard
+Generally, pressing `tab` moves focus between elements and pressing `enter` will interact with them.
+
+If you are using Safari, `option-tab` is the default keyboard shortcut to move focus between elements. You can update your settings to make `tab` the default navigation key by following the instructions in this article: [Quick Tip: Safari Keyboard Navigation](https://a11yproject.com/posts/safari-keyboard-navigation/)
+
+Additional keyboard shortcuts are described here: [WebAIM - Keyboard Accessibility](https://webaim.org/techniques/keyboard/)
+
+### Things to keep in mind while testing
```
Comment (scottaohara) [inline _posts/2013-01-11-navigate-using-just-your-keyboard.md]: `h2` not `h3`

File: _posts/2013-01-11-navigate-using-just-your-keyboard.md [hunk 6]
```diff
@@ -9,11 +9,26 @@ categories:
...
+It is not enough to simply tab through a webpage from start to end. While navigating the site, keep these things in mind:
+- Interactive elements should be functional using only keyboard shortcuts.
+- The focused element should be visible and have an obvious focus style. See article: ["Quick Tip: Never remove CSS outlines"](https://a11yproject.com/posts/never-remove-css-outlines/).
+- Focus should move between elements as they appear on the page and not jump back and forth.
+- There should be a way to skip past lengthy navigation items. See article: ["How–to: Use Skip Navigation links"](https://a11yproject.com/posts/skip-nav-links/).
+
+### Further Reading
```
Comment (scottaohara) [inline _posts/2013-01-11-navigate-using-just-your-keyboard.md]: `h2` not `h3`

File: _posts/2013-01-11-navigate-using-just-your-keyboard.md [hunk 7]
```diff
@@ -9,11 +9,26 @@ categories:
...
+### How to navigate a website with a keyboard
+Generally, pressing `tab` moves focus between elements and pressing `enter` will interact with them.
+
+If you are using Safari, `option-tab` is the default keyboard shortcut to move focus between elements. You can update your settings to make `tab` the default navigation key by following the instructions in this article: [Quick Tip: Safari Keyboard Navigation](https://a11yproject.com/posts/safari-keyboard-navigation/)
+
+Additional keyboard shortcuts are described here: [WebAIM - Keyboard Accessibility](https://webaim.org/techniques/keyboard/)
+
+### Things to keep in mind while testing
+It is not enough to simply tab through a webpage from start to end. While navigating the site, keep these things in mind:
+- Interactive elements should be functional using only keyboard shortcuts.
```
Comment (scottaohara) [inline _posts/2013-01-11-navigate-using-just-your-keyboard.md]: i don't think "shortcut" is the right word here.

maybe 
"Interactive elements should be functional using their default activation keys. e.g. <kbd>Enter</kbd> for links and <kbd>Space</kbd> for buttons."

as an example

File: _posts/2013-01-11-navigate-using-just-your-keyboard.md [hunk 8]
```diff
@@ -9,11 +9,26 @@ categories:
...
+- The focused element should be visible and have an obvious focus style. See article: ["Quick Tip: Never remove CSS outlines"](https://a11yproject.com/posts/never-remove-css-outlines/).
+- Focus should move between elements as they appear on the page and not jump back and forth.
+- There should be a way to skip past lengthy navigation items. See article: ["How–to: Use Skip Navigation links"](https://a11yproject.com/posts/skip-nav-links/).
```
Comment (scottaohara) [inline _posts/2013-01-11-navigate-using-just-your-keyboard.md]: not necessarily just navigation items, but lengthy spans of global content.

**Comment by missmatsuko** [comment]
Thanks, @scottaohara. My edits have been added - let me know if there's any other changes to make!

**Comment by scottaohara** [comment]
Thank you @missmatsuko.  Will review

File: _posts/2013-01-11-navigate-using-just-your-keyboard.md [hunk 9]
```diff
@@ -9,11 +9,25 @@ categories:
...
+Additional keyboard shortcuts are described here: [WebAIM - Keyboard Accessibility](https://webaim.org/techniques/keyboard/)
+
+## Things to keep in mind while testing
+It is not enough to simply tab through a webpage from start to end. While navigating the site, keep these things in mind:
+- Interactive elements should be functional using only keyboard input.
+- The focused element should be visible and have an obvious focus style. See article: ["Quick Tip: Never remove CSS outlines"](https://a11yproject.com/posts/never-remove-css-outlines/).
+- Focus should move between elements as they appear on the page and not jump back and forth.
+- There should be a way to skip past lengthy spans of content, like navigation menus. See article: ["How–to: Use Skip Navigation links"](https://a11yproject.com/posts/skip-nav-links/).
+
+### Further Reading
```
Comment (scottaohara) [inline _posts/2013-01-11-navigate-using-just-your-keyboard.md]: Again, is this exclusively a sub-section of "things to keep in mind while testing"?

**Comment by scottaohara** [comment]
Thanks @missmatsuko 

Going to leave this open a bit in case someone else has some feedback for you, but I think this is looking pretty good.

Thanks again

**Comment by ericwbailey** [comment]
Looks good to me, too! Merging!

**Pull request merged**